src/ngrest/render/RenderCrud.php

```diff
@@ -365,6 +365,102 @@ public function getCompositionKeysForButtonActions($modelPrefix)
         return "[". implode(",", $output) . "]";
     }
     
+    /**
+     * Get the ng-show condition for a given button
+     *
+     * Evaluates the ng-show condition from a list context. A condition like
+     * `{field} == true` would return `item.field == true`.
+     * 
+     * Conditions needs to be defined in the model's ngRestScopes() along with fields definition
+     * indexed by buttonCondition and provide as a string or a [field=>value] array.
+     * ```php
+     * public function ngRestScopes()
+     * {
+     *     return [
+     *       ['update', ['field1', 'field2', 'field3'], ['buttonCondition' => ['{created_by}' => Yii::$app->adminuser->id]] ]
+     *     ]
+     * } 
+     * ```
+     * 
+     * 
+     * Conditions may be defined also in the model ngRestConfigOptions() along with other
+     * options
+     * Exemple :
+     *
+     * ```php
+     *    public function ngRestConfigOptions()
```

Comment:
This method `ngRestConfigOptions`does not exists i think you mean ngRestScopes()

I also think its important to note that if multiple conditions are used its an AND connection.

Reply:
I am referring to 
https://github.com/luyadev/luya-module-admin/blob/68eb1a59b23be90deda9553f159e580d1a5c342c/src/ngrest/base/NgRestModel.php#L831
used to set the option propert or pointer of the Config.